.github/workflows/staticcheck.yaml

```
@@ -20,5 +21,7 @@ jobs:
         run: yarn prettier:check
       - id: run-test
         run: yarn test:ci
-      - id: codecov-script
-        run: ./scripts/codecov.sh
+      - id: codecov-script        
+        uses: codecov/codecov-action@84508663e988701840491b86de86b666e8a86bed # v4.3.0, needs token
```

```suggestion
        uses: codecov/codecov-action@84508663e988701840491b86de86b666e8a86bed # v4.3.0
```

Maybe there's no need to specify since we now have the token defined here?

---

Yep! Done in https://github.com/pagopa/io-app/pull/5724/commits/34ffeafb050a07ac56f6143f3d3c03e14579be92